pkgs/misc/cups/drivers/samsung/4.01.17.nix

```diff
@@ -74,7 +74,12 @@ in stdenv.mkDerivation rec {
     cp -r ./* $out/share/cups/model/samsung
   '';
 
-  ppdFileCommands = [ "pstosecps" "pstospl" "rastertospl" ];
+  ppdFileCommands = [
+    "pstosecps"
+    "pstospl"
+    "pstosplc"
+    "rastertospl"
```

Comment:
I usually try to separate commits if their changes are conceptually different, to facilitate bisection if a bug is introduced.  Here, the first commit should not affect the package's output at all (besides its hash), while the second commit intentionally alters the output, as it fixes an oversight that was already present for a long time.  I think the separation into two commits is justified here, what do you think?

Reply:
> I usually try to separate commits if their changes are conceptually different, to facilitate bisection if a bug is introduced. [...] I think the separation into two commits is justified here, what do you think?

I agree with this justification for separating / merging commits. Still I would have merged these two commits because I wouldn't have had the patience to vouch for the previous change to not change the resulting ppd files, and only change the inputs. But that's me, you are much more thorough then me! :)